docker/dockerdefaults.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# vim: ft=yaml sw=2
```

Comment:
Why is this here?

Reply:
AFAIK most editors parse this header, wanted to mark the width at 2 spaces as in your other file. But I believe it can also be infered, so I'll remove it.